app/controllers/api/v1/members_controller.rb

```diff
@@ -26,14 +26,32 @@ def update
 
     api :DELETE, '/organizations/:organization_id/members/:id', 'Destroy member'
     def destroy
-      authorize @member
       @member.destroy
     end
 
+    api :GET, '/organizations/:organization_id/members/:id', 'Return member'
+    def show
+    end
+
+    api :GET, '/organizations/:organization_id/member_info', 'Return current member'
+    def current
+      @member = current_member
+    end
+
+    api :PUT, '/organizations/:organization_id/last_visit', 'Update last viewed at for current member'
+    def update_last_viewed_at
+      if current_member.update(last_visited_at: Time.current)
```

Comment:
@yunixon add ability to pass `last_visited_at` as param, but if it's not passed, then use `Time.current`